bin/setup.dart

```diff
@@ -28,13 +29,40 @@ void extract(String fileName, [String dir = '']) {
   }
 }
 
+/// Places downloaded mobile binaries to proper location.
+void placeMobileBinaries(String platform, String fileName) {
+  final android =
+      '${findPackageRoot().toFilePath(windows: Platform.isWindows)}/android';
```

Comment:
1. You don't have to write `windows: Platform.windows` since this method will automatically detect the platform.
2. Instead of joining path using a `/` you should use the `join` method from `path` package.
You should do these changes in other places as well..